### ISSUE TITLE
Выделяемые из текста цепочки слов

### ISSUE THREAD

**Issue by KotIva** [yandex/tomita-parser#142]
Здравствуйте. Столкнулся с необходимостью рассмотреть 2 предложения разделенные знаком переноса строки как одну. Подскажите пожалуйста, есть ли способ это сделать? Пытался добавить в строку подобную конструкцию : (EOSent), но это не сработало. И если кто знает - можете поподробнее рассказать про этот  терминал EOSent, принцип работы и использования?

**Comment by victorbocharov** [comment]
EOSent нужен для того, чтобы правило срабатывало на конце предложения. Подразумевается, что после EOSent уже ничего не сработает, т.к. дальше токенов в предложении нет.

Приведите пример двух предложений, которые не склеились.

**Comment by KotIva** [comment]
![Пример исходного текста](https://user-images.githubusercontent.com/85670004/121647946-a1e9d880-ca9f-11eb-83d9-0a5065695f03.JPG)
![Тот же текст, только уже разобранный в файле PrettyOutput](https://user-images.githubusercontent.com/85670004/121648148-d8bfee80-ca9f-11eb-87af-98deb3a1206d.JPG)

**Comment by victorbocharov** [comment]
Конец предложения после аббревиатур - это частотная проблема. Она решается добавлением словаря аббревиатур. См. https://github.com/yandex/tomita-parser/issues/46

Двойной перенос строки считается концом предложения. Так было задумано. Если это неудобно, то лучше сделать препроцессинг и убрать двойной перенос строки или даже лишние точки. Длинные предложения, которые получатся после удаления точек, на некоторых грамматиках могут увеличить время работы парсера.

**Comment by KotIva** [comment]
Спасибо за ответ. А есть где-либо расширенная документация по Томите ну или не подскажите где можно найти более тонкие аспекты Томиты, чем те, которые описаны в официальной документации ?

**Comment by victorbocharov** [comment]
Нет, тайной внутренней документации нет. Большинство типовых вопросов есть в тикетах. Остальное - в коде. Если не находите ответа на свой вопрос, то открывайте новый тикет.

**Comment by KotIva** [comment]
Понял. Спасибо.
